skopt/searchcv.py

```diff
@@ -398,6 +399,146 @@ def add_spaces(self, names, search_spaces):
         for space, name in zip(search_spaces, names):
             self.search_spaces_[name] = space
 
+    # copied for compatibility with 0.19 sklearn from 0.18 BaseSearchCV
+    @property
+    def best_score_(self):
```

Comment:
Actually these both are used in the code and examples, hence were breaking the circle ci